recipes/lambda/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: lambda
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/lambda/archive/lambda-v{{ version }}.tar.gz
+  sha256: 324f90dbc095c56792a77a002e64a98465a4853f74a141a1182e7a6f540545a2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gxx_linux-64 # [linux]
+    - clangxx_osx-64 # [osx]
+    - cmake >=3.0.0
```

Comment:
build only needs to contain compile time dependencies that are not necessary at runtime. Host is for compile-time dependencies that are also necessary at runtime. Hence, bzip2 and zlib only need to be added to host.

Reply:
Thanks! I had only tested removing the dependencies completely but hadn't grasped that concept. It's clearer now, and works... ;)